app/src/main/java/com/hedvig/app/feature/embark/EmbarkViewModel.kt

```diff
@@ -501,7 +523,7 @@ abstract class EmbarkViewModel(
 
 class EmbarkViewModelImpl(
     private val embarkRepository: EmbarkRepository,
-    tracker: EmbarkTracker,
+    tracker: EmbarkTracker
```

Comment:
```suggestion
    tracker: EmbarkTracker,
```

We should check your code style settings, it's weird that it's eating the trailing commas 😧